src/app/globals.css

```diff
@@ -27,9 +27,13 @@
     @apply text-lg;
```

Comment:
I have a question, but what is the point of making the default font size 18px (text-lg), not 16px (text-base)? Was it something the designers have already decided?

Reply:
It looks like the default font size used in Figma, was actually 18px. That's why it is.